datasets/nell/README.md

```diff
@@ -0,0 +1,234 @@
+Learn to read better than yesterday. NELL uses a variety of methods to extract beliefs from the web. These are retrained, using the growing knowledge base as a self-supervised collection of training examples. The result is a semi-supervised learning method that couples the training of hundreds of different extraction methods for a wide range of categories and relations. Much of NELL’s current success is due to its algorithm for coupling the simultaneous training of many extraction methods.
+
+For more information, see: http://rtw.ml.cmu.edu/rtw/resources
+
+### Languages
+en, and perhaps some others
+
+## Dataset Structure
+
+### Data Instances
+
+There are four configurations for the dataset: nell_belief, nell_candidate, nell_belief_sentences, nell_candidate_sentences. 
+
+nell_belief and nell_candidate defines:
+
+``
+{
+	'entity': ...,
+	'relation': ...,
+	'value': ...,
+	'iteration_of_promotion': ...,o
+	'probability': ...,
+	'entity_literal_strings': ...,
+	'value_literal_strings': ...
+	'best_entity_literal_string': ...
+	'best_value_literal_string': ...
+	'categories_for_entity': ...
+	'categories_for_value': ...
+	'candidate_source': ...
+}
+``
+
+nell_belief_sentences, nell_candidate_sentences defines:
+
+``
+{
+	'entity': ...,
+	'relation': ...,
+	'value': ...,
+	'sentence': ...,
+	'probability': ...,
+	'url': ...
+	'count': ...
+	'sentence_type': ...
+}
+``
+
+### Data Fields
+
+For nell_belief and nell_canddiate configurations. From http://rtw.ml.cmu.edu/rtw/faq:
+* entity: The Entity part of the (Entity, Relation, Value) tripple. Note that this will be the name of a concept and is not the literal string of characters seen by NELL from some text source, nor does it indicate the category membership of that concept
+* relation: The Relation part of the (Entity, Relation, Value) tripple. In the case of a category instance, this will be "generalizations". In the case of a relation instance, this will be the name of the relation.
+* value: The Value part of the (Entity, Relation, Value) tripple. In the case of a category instance, this will be the name of the category. In the case of a relation instance, this will be another concept (like Entity).
+* iteration_of_promotion: The point in NELL's life at which this category or relation instance was promoted to one that NELL beleives to be true. This is a non-negative integer indicating the number of iterations of bootstrapping NELL had gone through.
+* probability: A confidence score for the belief. Note that NELL's scores are not actually probabilistic at this time.

```

Comment:
maybe name this field `score` then ?

Reply:
"Probability" is the author's original name for this field. I think "score" is better though, so I can make this change